front/app/components/UI/FullscreenModal/index.tsx

```diff
@@ -22,16 +22,21 @@ import { isNilOrError } from 'utils/helperUtils';
 const slideInOutTimeout = 500;
 const slideInOutEasing = 'cubic-bezier(0.19, 1, 0.22, 1)';
 
-const Container = styled.div<{ windowHeight: number; zIndex?: number }>`
+const Container = styled.div<{
+  windowHeight: number;
+  zIndex?: number;
+  contentBgColor?: InputProps['contentBgColor'];
```

Comment:
Do we need to use it like this - `InputProps['contentBgColor']` ? I'm not always a fan of this since if the prop `contentBgColor` changes, this won't raise a type error right?

Reply:
Yeah, it does raise an error then! :)

<img width="1137" alt="image" src="https://github.com/user-attachments/assets/84b35dd7-fb74-4925-8c01-bbd1a365d136">

Reply:
Nice! Ok that's great then :+1: Thanks!

Reply:
Also, if you rename the prop via Rename Symbol, it'll update all these spots in one go.
<img width="573" alt="Screenshot 2024-11-19 at 10 32 15" src="https://github.com/user-attachments/assets/1ad288dc-e739-4775-84a1-24220a0d1902">